src/sorting.cpp

```diff
@@ -49,13 +52,11 @@ include_category *include_categories[kIncludeCategoriesCount];
  * @retval  > 0
  * @retval  < 0
  */
-static int compare_chunks(Chunk *pc1, Chunk *pc2, bool tcare = false);
+static int compare_chunks(Chunk *pc1, Chunk *pc2, bool tcare);
```

Comment:
why do you suppress the default value?

Reply:
`compare_chunks()` is only ever called once and the parameter is always passed so the default value is never used.